src/controllers/PetitionController.js

```diff
@@ -1,31 +1,54 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-useless-catch */
 import PetitonService from '../services/PetitionServices';
 import Util from '../utils/Utils';
+import db from '../models';
 
 const util = new Util();
 
 class PetitionController {
-    static async addPetition(req, res) {
-        const votes = 0;
-        const expired = false;
-        const { title, description } = req.body;
-        let user_id = 1; // get from the token
-        const newPetition = {
-            user_id, title, description, votes, expired
-        }
-        try {
-            const createdPetition = await PetitonService.addPetition(newPetition);
-            
-            if (createdPetition === 'exists') {
-                util.setError(409, `Petition with the title: '${title}' already exists`);
-                return util.send(res);
-            }
-            util.setSuccess(201, 'Petition successifully created', createdPetition);
-            return util.send(res);
-        } catch (error) {
-            util.setError(400, error.message);
-            return util.send(res);
-        }
+  static async addPetition(req, res) {
+    const votes = 0;
+    const expired = false;
+    const { title, description } = req.body;
+    const user_id = 1; // get from the token
+    const newPetition = {
+      user_id, title, description, votes, expired,
+    };
+    try {
+      const createdPetition = await PetitonService.addPetition(newPetition);
+
+      if (createdPetition === 'exists') {
+        util.setError(409, `Petition with the title: '${title}' already exists`);
+        return util.send(res);
+      }
+      util.setSuccess(201, 'Petition successifully created', createdPetition);
+      return util.send(res);
+    } catch (error) {
+      util.setError(400, error.message);
+      return util.send(res);
     }
+  }
 }
 
+// eslint-disable-next-line consistent-return
+export const deletePetition = async (req, res) => {
```

Comment:
Great work. However, you consider making the deletePetition function a method inside the PetitionController class in the next iteration.

Reply:
Ok, I'll do that next time